### PR TITLE
Ruby now uses HackerOne for managing security vulnerability reports

### PR DESCRIPTION
**Pull request by reedloden**
Update security documentation to point to https://hackerone.com/ruby.

### CODE REVIEW

**Comment by reedloden** [comment]
@tenderlove here you go :)

**Comment by hsbt** [comment]
We are coordinating to use HackerOne now. Please wait to merge this.

**Comment by shugo** [comment]
We should warn that only vulnerabilities of Ruby (not the website), should be reported to Hacker One, shouldn't we?

**Comment by zzak** [comment]
@shugo Yes, I think we should also clarify _what kind_ of vulnerabilities should be reported

**Comment by reedloden** [comment]
@shugo / @zzak -- I made a few changes to the text, but please feel free to provide additional language. I see you all have already made some changes to https://hackerone.com/ruby to note the scope and type of vulns.

File: en/security/index.md [hunk 1]
```diff
@@ -9,9 +9,22 @@ Here you will find information about security issues of Ruby.
...
-security@ruby-lang.org ([the PGP public key](/security.asc)), which is a
-private mailing list. Reported problems will be published after fixes.
+Security vulnerabilities in the Ruby programming language should be
+reported through our [bounty program page at
+HackerOne](https://hackerone.com/ruby). Please ensure you read the
+specific details around the scope of our program before reporting
+an issue. Any valid reported problems will be published after fixes.
+
+If you have found an issue affecting one of our websites, please
+report it [here](https://github.com/ruby/www.ruby-lang.org/issues/new).
```
Comment (stomar) [inline en/security/index.md]: `[here]` -> `[on GitHub]` or similar

File: en/security/index.md [hunk 2]
```diff
@@ -9,9 +9,22 @@ Here you will find information about security issues of Ruby.
...
+
+If you have found an issue that affects a specific Ruby gem, follow the
+instructions
+[here](http://guides.rubygems.org/security/#reporting-security-vulnerabilities).
```
Comment (stomar) [inline en/security/index.md]: `instructions [here]` -> `[instructions on RubyGems.org]`

**Comment by reedloden** [comment]
@stomar updated :)

**Comment by reedloden** [comment]
Anything else blocking this, or can we get this merged? (cc @hsbt)

**Comment by hsbt** [comment]
@reedloden Thank you for your update.

I confirmed your changes. It is enough for our report line. I appreciate it

**Pull request merged**